packages/react-components/src/components/styles/InputBoxComponent.style.ts

```diff
@@ -13,6 +13,7 @@ export const inputBoxWrapperStyle = mergeStyles({
  * @private
  */
 export const inputBoxStyle = mergeStyles({
+  overflow: 'auto',
```

Comment:
Please add a comment here that by default, the `TextField` sets `overflow` to `hidden`, so we need to override it.